src/Symfony/Component/DependencyInjection/Argument/TaggedIteratorArgument.php

```diff
@@ -19,16 +19,40 @@
 class TaggedIteratorArgument extends IteratorArgument
 {
     private $tag;
+    private $indexAttribute;
+    private $defaultIndexMethod;
 
-    public function __construct(string $tag)
+    /**
+     * TaggedIteratorArgument constructor.
```

Comment:
This should be removed. We removed useless doc in the past.

Reply:
I'll remove that in my new PR following this one, 
thanks